test/e2e/v3/generate_test.go

```diff
@@ -200,6 +200,9 @@ Count int `+"`"+`json:"count,omitempty"`+"`"+`
 	ExpectWithOffset(1, utils.UncommentCode(
 		filepath.Join(kbc.Dir, "config", "default", "kustomization.yaml"),
 		"#- ../prometheus", "#")).To(Succeed())
+	ExpectWithOffset(1, utils.UncommentCode(
+		filepath.Join(kbc.Dir, "config", "default", "kustomization.yaml"),
+		"#- manager_config_patch.yaml", "#")).To(Succeed())
```

Comment:
The e2e project wasn't scaffolded with `--component-config` so this patch will cause `kustomize build` conflicts.
```suggestion
```